modules/local/gffread.nf

```diff
@@ -0,0 +1,50 @@
+process GFFREAD {
+    tag "$gff"
+    label 'process_low'
+
+    conda "bioconda::gffread=0.12.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gffread:0.12.1--h8b12597_0' :
+        'biocontainers/gffread:0.12.1--h8b12597_0' }"
+
+    input:
+    tuple val(meta), path(gff)
+    tuple val(meta), path(fasta)
+
+    output:
+    path "*.gtf"        , emit: gtf,        optional: true
+    path "*.fa"         , emit: tr_fasta,   optional: true
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args   ?: ''
+    def genome = fasta ? "-g ${fasta}" : ''
+    def prefix = task.ext.prefix ?: gff.getSimpleName()
+    def outp   = args.contains("-w ") ? "" : "-o ${prefix}.gtf"
```

Comment:
high risk that you input will be named the same as the output

Reply:
I have made some changes, maybe not the best possible but it should work, if you have any idea on how to improve it, let me know